modules/tb-all-in-one/main.tf

```diff
@@ -0,0 +1,53 @@
+resource "digitalocean_volume" "storage" {
+  region      = "nyc1"
+  name        = "storage-${var.name}"
+  size        = 10
+  description = "${var.name} Volume"
+}
+
+resource "digitalocean_droplet" "droplet-1" {
+  image = "ubuntu-16-04-x64"
+  name = "${var.name}-1"
+
+  region = "${var.region}"
+  size = "${var.droplet_size}"
+
+  private_networking = true
+  ipv6 = true
+
+  monitoring = true
+
+  ssh_keys = "${var.ssh_keys}"
+
+  connection {
+    user = "root"
+    type = "ssh"
+    private_key = "${file(var.pvt_key)}"
```

Comment:
does.........this work? Are you pulling the private key from your local computer? from way outside your current directory? this feels weird.

Reply:
yup. https://www.digitalocean.com/community/tutorials/how-to-use-terraform-with-digitalocean did it, and I had started from it

Reply:
next question. what's a pivot key?